Sourcery/Parsing/FileParser.swift

```diff
@@ -155,7 +156,7 @@ final class FileParser {
             case .varParameter:
                 return parseParameter(source)
             default:
-                if verbose { print("\(logPrefix)Unsupported entry \"\(access) \(kind) \(name)\"") }
+                Log.warning("\(logPrefix) Unsupported entry \"\(access) \(kind) \(name)\"")
```

Comment:
I'd keep the distinction between verbose and warnings here, this is common as there are language concepts we might not support vs we should always be able to parse enum case